mautic/mautic
# TPROD-323 - PluginBundle - Migrate PHP templates to Twig templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



- `/s/plugins`
  - Page render correctly
  - Able to filter
  - Click on each plugin and ensure the modal displays and there are no errors with formatting
  - Click on **Zapier**, ensure the notification is rendered correctly
  - **Known Issue**: On the 5.x branch, as well as this one, for plugins that have a Contact Mapping field, it will persist if a required field is not mapped.
- Auth/Reauth App
  - Connect one of the plugins, example Facebook, to an app that you created.
